### ISSUE TITLE
LOAD VIDEO deletes video from table when run a second time

### ISSUE THREAD

**Issue by Aryan-Rajoria** [georgia-tech-db/eva#531]
Adding a video with command
`"LOAD VIDEO '<file_name>' INTO <table_name>"`
gives success code, and loads video properly
However, running the same command again causes the table to lose its contents.
Example:
```
file_name = '/workspace/soft/data/sample/eva/kadal.mp4'
cursor.execute(f"LOAD VIDEO '{file_name}' INTO v1")
result = cursor.fetch_all()
print(result)
```
output:
```
@status: ResponseStatus.SUCCESS
@batch: 
                             0
0  Number of loaded videos: 1
@query_time: 0.09101645499868027
```
```
cursor.execute("SELECT id FROM v1")
result = cursor.fetch_all()
print(result)
```
output:
```
@status: ResponseStatus.SUCCESS
@batch: 
       v1.id
0         0
1         1
2         2
3         3
4         4
...     ...
4182   4182
4183   4183
4184   4184
4185   4185
4186   4186

[4187 rows x 1 columns]
@query_time: 1.6801937140007794
```
```
cursor.execute(f"LOAD VIDEO '{file_name}' INTO v1")
result = cursor.fetch_all()
print(result)
cursor.execute("SELECT id FROM v1")
result = cursor.fetch_all()
print(result)
```
output:
```
@status: ResponseStatus.FAIL
@batch: 
 None
@error: Load video failed: encountered unexpected error Duplicate File: /workspace/soft/data/sample/eva/kadal.mp4 already exists in the table v1
@status: ResponseStatus.SUCCESS
@batch: 
 Empty DataFrame
Columns: []
Index: []
@query_time: 0.015552428001683438
```

**Comment by gaurav274** [comment]
#532

**Comment by gaurav274** [comment]
Fixed iwth #532
